contracts/vaults/RibbonThetaVault.sol

```diff
@@ -459,16 +526,53 @@ contract RibbonThetaVault is DSMath, GnosisAuction, OptionsVaultStorage {
         uint256 shortAmount = wmul(freeBalance, lockedRatio);
         lockedAmount = shortAmount;
 
-        GammaProtocol.createShort(
+        GAMMA_PROTOCOL.createShort(
             GAMMA_CONTROLLER,
             MARGIN_POOL,
             newOption,
             shortAmount
         );
 
+        startAuction();
+
         emit OpenShort(newOption, shortAmount, msg.sender);
     }
 
+    /**
+     * @notice Initiate the gnosis auction.
+     */
+    function startAuction() public onlyManager {
+        GnosisAuction.AuctionDetails memory auctionDetails;
+
+        auctionDetails.oTokenAddress = currentOption;
+        auctionDetails.asset = asset;
+        auctionDetails.underlying = underlying;
+        auctionDetails.manager = manager;
+        auctionDetails.premiumDiscount = premiumDiscount;
+        auctionDetails.duration = 6 hours;
+
+        GnosisAuction.startAuction(
+            GNOSIS_EASY_AUCTION,
+            optionsPremiumPricer,
+            auctionDetails
+        );
+    }
+
+    /**
+     * @notice Burn the remaining oTokens left over from gnosis auction.
+     */
+    function burnRemainingOTokens() external onlyManager nonReentrant {
```

Comment:
This process should be bundled together with closing the auction imo, so it would be like:

```solidity
function closeAuction() {
  GnosisAuction.settleAuction()
  burnRemainingOTokens()
}
```

Reply:
I'm okay with the change being a follow-up PR because this PR is getting huge

Reply:
Yeah, that is how I originally designed it. I decoupled to save gas. can do an offchain check to see if we even _need_ to burn remaining oTokens. but on second thought, perhaps convencience of one tx >> if statement gas

Reply:
Yea let's stick with 2 different functions, 1 for closing the auction, another for burning otokens.

Reply:
closing the auction = IERC20(GNOSIS_EASY_AUCTION).settleAuction(id)

we can just call this function directly on gnosis contract, don't think we should have a proxy for it in theta vault code